openpype/modules/kitsu/utils/credentials.py

```diff
@@ -102,3 +104,14 @@ def set_credentials_envs(login: str, password: str):
     """
     os.environ["KITSU_LOGIN"] = login
     os.environ["KITSU_PWD"] = password
+
+
+def emit_on_kitsu_login(login: str):
+    """Notifies listeners that Kitsu module succesfully connected,
+    and passes them data
+
+    Args:
+        login (str): Kitsu username
+    """
+    event_data = {"username": login}
+    emit_event("kitsu.user.logged", data=event_data, source="kitsu")
```

Comment:
Is there a reason we want this as a separate function?

I feel like adding this above into the login function is just as clear? Or are there reasons to call this function outside of the login function?
```python
emit_event("kitsu.user.logged", 
           data={"username": login}, 
           source="kitsu")
```

Reply:
It just felt more natural to me that way but now that I think of it, I agree with you that it would be simpler to make it part of the login function. Should I make that change?

Reply:
> EDITED: Waiting with merge to this https://github.com/pypeclub/OpenPype/pull/3785#discussion_r961702993

I think so @Sharkitty - yes please.